math/lcm.go

```diff
@@ -0,0 +1,49 @@
+// Formula from https://en.wikipedia.org/wiki/Least_common_multiple
+// Find the least common multiple of two integers
+// by using the greatest common divisor.
+
+package main
+
+import (
+	"fmt"
+)
+
+// abs: absolute value for an integer
+func abs(a int) int {
+	if a < 0 {
+		return -a
+	}
+	return a
+}
+
+// gcd: greatest common divisor -
+// the largest positive integer that divides both a and b
+func gcd(a, b int) int {
+	if b == 0 {
+		return a
+	}
+	return gcd(b, a%b)
+}
+
+// lcm: least common multiple -
```

Comment:
remove the `:` after the function name. Also, you should capitalise the first letter of the main algorithms that you are submitting. In this case, the `lcm` is the main function that you implemented, therefore I suggest

```suggestion
// Lcm least common multiple -
```